docs/api/structures/client-request-constructor-options.md

```diff
@@ -0,0 +1,42 @@
+# ClientRequestConstructorOptions Object
```

Comment:
We probably don't want to do this? This constructor object should be automatically generated by the docs parser, if it's not then that's a bug in the docs parser we should fix

Reply:
Agreed. I think this is resolved in docs-parser 0.4.2 in that when there's a union (here, an object | string) followed by a key list, docs-parser now only applies the list to the first union type.